roles/yoda_portal/defaults/main.yml

```diff
@@ -13,6 +13,9 @@ update_portal: true
 # Yoda theme
 yoda_theme_path: /var/www/yoda/themes              # Base path holding customised portal themes
 yoda_theme: uu                                     # Yoda theme: uu or vu (default: uu)
+yoda_main_path: /var/www/yoda                      # Yoda portal installation path
+theme_mapping: {"uu": "Utrecht University - default", "uu_its": "Utrecht University - ITS", "uu_gw": "Utrecht University - Humanities", "uu_youth": "Utrecht University - YOUth", "uu_i-lab": "Utrecht University - i-lab", "uu_science": "Utrecht University - Science", "uu_fsw": "Utrecht University - Social Science", "uu_geo": "Utrecht University - Geo", "uu_dgk": "Utrecht University - Veterinary Medicine", "uu_dag": "Utrecht University - Data Archive for Geosciences (DAG)", "vu": "Vrije University Amsterdam", "wur": "Wageningen University & Research"}
```

Comment:
I added the theme_mapping here, but I wonder if this is necessary given allinone.yml already has the same variable

Reply:
Maybe the default theme mapping here should just have the main uu theme? Or the main uu theme, the vu, and the wur?